src/components/SchedulePanel/components/RemoveHoursDialog/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Fragment } from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogActions from '@material-ui/core/DialogActions';
+import Button from '@material-ui/core/Button';
+import { gordonColors } from '../../../../theme';
+
+export default class RemoveHoursDialog extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      removeOfficeHoursOpen: false,
+    };
+    this.handleRemoveOfficeHoursClose = this.handleRemoveOfficeHoursClose.bind(this);
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    this.props.onDialogSubmit();
+    console.log('Happened');
```

Comment:
This console log is really not useful. Can we either remove it, or, if it is necessary, give it a more descriptive message?

Reply:
Change made